aesara/tensor/random/op.py

```diff
@@ -399,3 +400,36 @@ def grad(self, inputs, outputs):
 
     def R_op(self, inputs, eval_points):
         return [None for i in eval_points]
+
+
+class AbstractRNGConstructor(Op):
+    def make_node(self, seed=None):
+        if seed is None:
+            seed = NoneConst
+        else:
+            seed = as_tensor_variable(seed)
+        inputs = [seed]
+        outputs = [self.random_type()]
+        return Apply(self, inputs, outputs)
+
+    def perform(self, node, inputs, output_storage):
+        (seed,) = inputs
+        if seed is not None and seed.size == 1:
+            seed = int(seed)
```

Comment:
Had to do this because of what seems to be a bug in Numpy

```python
import numpy as np

x = np.random.default_rng(3) #Works
x = np.random.default_rng(np.array([3])) #Works
x = np.random.default_rng(np.array(3)) #Throws error
```